docs/src/howto-guides/017_apama_software_management_plugin.md

```diff
@@ -88,3 +88,15 @@ TedgeDemoMonitor                                             1
 ```
 
 You can find more information on this test Apama project [here](https://github.com/thin-edge/thin-edge.io_examples/tree/main/StreamingAnalytics#testing-a-project).
+
+## Filter packages by name and maintainer
+
+You can filter the list output using two filtering criteria: name and maintainer. Create the `apt` table in `tedge.toml` and fill it with the `name` and `maintainer` keys. The value of each filter key should be a **valid regex pattern**. Your filters should look like this:
+
+```toml
+[apt]
+name = "exemplary_name.*"
+maintainer  = "exemplary_maintainer"
+```  
```

Comment:
Can't these be set/unset using the `tedge cli`?

Reply:
No this will be addressed once merged the new tedge-config PR.